bleak/backends/corebluetooth/characteristic.py

```diff
@@ -63,7 +63,7 @@ def __init__(self, obj: CBCharacteristic):
         # self.__props = obj.properties()
         self.__props: List[str] = [
             _GattCharacteristicsPropertiesEnum[v][0]
-            for v in [2 ** n for n in range(10)]
+            for v in [2**n for n in range(10)]
```

Comment:
It would be nice to put unrelated black format changes in a separate commit.

Reply:
Actually, I'll just go ahead and do that, then you can just rebase on the develop branch.